### PR TITLE
[postgres] Add functions to get the bounds of primary keys

### PR DESCRIPTION
**Pull request by nathan-artie**

### CODE REVIEW

File: lib/postgres/schema/schema.go [hunk 1]
```diff
@@ -204,3 +205,47 @@ func GetPrimaryKeys(db *sql.DB, schema, table string) ([]string, error) {
 	}
 	return primaryKeys, nil
 }
+
+func GetPrimaryKeysLowerBounds(db *sql.DB, schema, table string, primaryKeys []string, castedPrimaryKeys []string) ([]interface{}, error) {
+	minValues := make([]interface{}, len(primaryKeys))
```
Comment (nathan-artie) [inline lib/postgres/schema/schema.go:210]: @Tang8330 do we need to be making two slices here, seems like one would suffice?

Comment (nathan-artie) [reply lib/postgres/schema/schema.go:210]: Ohh I see, one is a slice of pointers.

File: lib/postgres/schema/schema_test.go [hunk 2]
```diff
@@ -126,3 +126,26 @@ func TestParseColumnDataType(t *testing.T) {
 		assert.Equal(t, testCase.expectedOpts, opts, testCase.name)
 	}
 }
+
+func TestSelectTableQuery(t *testing.T) {
```
Comment (nathan-artie) [inline lib/postgres/schema/schema_test.go:130]: Just moved this.

File: lib/postgres/schema/schema.go [hunk 3]
```diff
@@ -204,3 +204,68 @@ func GetPrimaryKeys(db *sql.DB, schema, table string) ([]string, error) {
...
+
+type selectTableQueryArgs struct {
+	Keys       []string
+	Schema     string
+	TableName  string
+	OrderBy    []string
+	Descending bool
+}
+
+func selectTableQuery(args selectTableQueryArgs) string {
```
Comment (nathan-artie) [inline lib/postgres/schema/schema.go:216]: Just moved this function, no changes.

**Review by Tang8330 — APPROVED** [review]

**Pull request merged**